README.md

```diff
@@ -82,6 +83,10 @@ for `some-exception`:
 By default an error is logged when an import could not be resolved, but it does not fail babel compilation.
 Setting this option to true will fail babel compilation with details.
 
+### resolveDirectories
+
+Directories which should be search for resolving modules. Defaults to node_modules.
```

Comment:
Can you mention priority in case of duplicates?  IE `resolveDirectories: ['bower_components', 'node_modules']` if both have `some-component` then I assume it'll resolve `bower_components/some-component/index.js` per order of array elements?

What happens if `node_modules/module2` imports `module1` but module1 exists in bower_components and node_modules?  Does it jump from node_modules to bower_components per the resolve order?  If this is the case we probably cannot fix it but might be worth having a warning about handling of duplicate modules / recommend avoiding duplicates?  Maybe we need a test for this too?  In this case I'd probably recommend that `node_modules` always be first when adding multiple `resolveDirectories`?


Also please backtick and quote node_modules, follow the wording used to tell of other defaults:
```
Default `'node_modules'`.
```

Reply:
Yes the array is the priority regardless of the import source. 

Will make the adjusments